docs/pages/application-access/guides/dynamodb.mdx

```diff
@@ -1,22 +1,24 @@
 ---
-title: Database Access with AWS DynamoDB
-description: How to access AWS DynamoDB with Teleport.
+title: Access AWS DynamoDB as an Application
```

Comment:
Do you think the reader will know what _as an Application_ means?

Reply:
I don't know myself. I'd welcome any suggestions on how to better differentiate this guide's title from the new one. As discussed in the comment above, the titles should be unique.

Reply:
"Access AWS DynamoDB via App Access"?

Reply:
that may be a bit of redundant phrasing tho

Reply:
"AWS DynamoDB using App Access"?

Reply:
See 0fdbfce9c6a73ad77d11b7701b36d4c7ffc73f7d